### PR TITLE
scale up rc is commented for patch rc

### PR DESCRIPTION
**Pull request by msenmurugan**
scale up rc is commented for patch rc. The scale up will take care in next reconciler event.

Ensure the replication controller pods are zero for upgrade scenario

### CODE REVIEW

File: pkg/util/common.go [hunk 1]
```diff
@@ -1406,10 +1411,10 @@ func PatchReplicationController(clientset *kubernetes.Clientset, old corev1.Repl
 		return err
 	}
 
-	newRc, err = PatchReplicationControllerForReplicas(clientset, newRc, new.Spec.Replicas)
-	if err != nil {
-		return err
-	}
+	// newRc, err = PatchReplicationControllerForReplicas(clientset, newRc, new.Spec.Replicas)
```
Comment (rrati) [inline pkg/util/common.go]: Remove commented out code

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/920#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/920#pullrequestreview-295084495" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/920#issuecomment-536636485" title="LGTM">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/920#" title="Author self-approved">msenmurugan</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/920#pullrequestreview-295084495" title="Approved">rrati</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.8.x/OWNERS)~~ [jdartigalongue,msenmurugan,rrati]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: new pull request created: #921

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/920#issuecomment-536637598):

>/cherrypick release-2019.10.x


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by operator-r2d2** [comment]
@msenmurugan: new pull request created: #922

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/920#issuecomment-536637673):

>/cherrypick saas-2019.8.4


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**